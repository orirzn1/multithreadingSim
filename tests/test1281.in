L8
S8
O11
N5

T0
I@0x00000000
LOAD $3, $7, $0
STORE $0, $1, $4
SUB $2, $2, $1
STORE $7, $2, 0x4
SUB $2, $4, $0
LOAD $2, $4, $0
SUB $3, $3, $1
SUBI $6, $2, 0x1
HALT $0

T1
I@0x00000000
LOAD $0, $3, 0x0
SUB $0, $0, $5
LOAD $5, $6, $5
ADDI $0, $3, 0x2
SUB $4, $5, $2
ADD $1, $3, $5
SUBI $3, $6, 0x1
STORE $4, $5, 0x0
STORE $2, $7, 0x4
ADD $7, $4, $3
HALT $0

T2
I@0x00000000
STORE $2, $1, $2
SUB $1, $7, $4
STORE $7, $3, 0x0
LOAD $1, $7, 0x4
LOAD $6, $2, 0x4
STORE $4, $1, 0x1
LOAD $6, $3, 0x3
STORE $3, $2, $0
HALT $0

T3
I@0x00000000
ADD $2, $3, $0
ADDI $0, $5, 0x1
ADD $2, $2, $0
LOAD $7, $6, 0x1
ADDI $4, $5, 0x2
HALT $0

T4
I@0x00000000
ADDI $7, $2, 0x3
ADDI $2, $5, 0x2
SUB $3, $0, $0
ADD $1, $2, $6
SUBI $3, $2, 0x4
ADD $7, $1, $1
ADDI $5, $3, 0x1
HALT $0

D@0x0
0x1
0x6
0x1
0x8
0x9
0x5
0x1
0x5
0x3
0x1
