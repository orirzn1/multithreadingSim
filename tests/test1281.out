
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x1	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFF	R7 = 0x0
Register file thread id 1:
	R0 = 0x2	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFF	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFF
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x1	R1 = 0x0	R2 = 0x1	R3 = 0x0	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x2	R2 = 0x2	R3 = 0xFFFFFFFE	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0x4
Blocked MT CPI for this program 5.906977

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFF	R7 = 0x0
Register file thread id 1:
	R0 = 0x2	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFF	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFF
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x1	R1 = 0x0	R2 = 0x1	R3 = 0x0	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x2	R2 = 0x2	R3 = 0xFFFFFFFE	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0x4
Finegrained Multithreading CPI for this program 1.627907

